addon/components/quarterly-closure/disabled-form-info.hbs

```diff
@@ -1,4 +1,2 @@
-<div uk-alert class="uk-alert-warning">
-  <FaIcon @icon="circle-info" @fixedWidth={{true}} class="margin-right-xs" />
-  {{t "ember-gwr.quarterlyClosure.components.disabledFormInfo"}}
-</div>
+<FaIcon @icon="circle-info" @fixedWidth={{true}} class="margin-right-xs" />
```

Comment:
Please ensure that the text and icon are middle aligned vertically

Reply:
Due to its small size i think no `flex` is needed. Would you still add it? I think that we use flex sometimes a bit too much for stuff like this.

Reply:
![alert](https://user-images.githubusercontent.com/32454507/190587076-48c9af42-6cfd-4bf0-9e3a-d8c15ffe0554.png)

It looks slightly shifted though :sweat_smile:

Reply:
Is this in ng?

Reply:
yes

Reply:
:cursing_face:

Reply:
Done

Reply:
:sweat_smile: